frontend/views/containers/group-settings/InvitationLinkModal.vue

```diff
@@ -6,7 +6,7 @@ modal-template(ref='modal' :a11yTitle='L("Add new members")')
   .c-container
     template(v-if='link')
       i18n.is-title-4(tag='h3') Share this link to grant access to your group.
-      i18n.has-text-1(tag='p') After the onboarding period has ended, everyone will be asked to vote on whether or not a new member should be added. But for now, enjoy 60 free passes!
+      i18n.has-text-1(tag='p') After the onboarding period has ended, everyone will be asked to vote on whether or not a new member should be added. But for now, enjoy 150 free passes!
```

Comment:
Could you make this a variable so that we don't have to re-translate it next time it changes?

Reply:
I jumped too early to making a fix and missed that part in the issue. updated.